app/models/vita_partner_zip_code.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: vita_partner_zip_codes
+#
+#  id              :bigint           not null, primary key
+#  zip_code        :string           not null
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  vita_partner_id :bigint           not null
+#
+# Indexes
+#
+#  index_vita_partner_zip_codes_on_vita_partner_id               (vita_partner_id)
+#  index_vita_partner_zip_codes_on_zip_code_and_vita_partner_id  (zip_code,vita_partner_id) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (vita_partner_id => vita_partners.id)
+#
+class VitaPartnerZipCode < ApplicationRecord
+  belongs_to :vita_partner
+  validate :record_of_zip_code
+  validates :zip_code, uniqueness: { scope: :vita_partner }
```

Comment:
Awesome, so this means that multiple vita partners can service the same zip code but that each VP can only have one entry for each zip code, right? That's awesome.

Reply:
Yes exactly! I added another test in the scenario where there is a dupe zip code record with different VP to make this clearer

Reply:
Note after talking Kelly & Jenny: should not be more than zip code for different VP